<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1121</width>
    <height>901</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QAxWidget" name="axWidget">
    <property name="geometry" stdset="0">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>581</width>
      <height>491</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn1">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>120</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>确定</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit1">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>80</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label1">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>10</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>显示出租车轨迹点</string>
    </property>
   </widget>
   <widget class="QLabel" name="label2">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>50</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>输入车号(1-10357)：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label3">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>170</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>矩形区域1：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit2">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>200</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>116.3</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit3">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>200</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>39.89</string>
    </property>
   </widget>
   <widget class="QLabel" name="label4">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>200</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>左上角经度：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label5">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>200</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>纬度：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit4_2">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>290</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label6">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>290</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>时 --</string>
    </property>
   </widget>
   <widget class="QLabel" name="label4_2">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>240</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>右下角经度：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label5_2">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>240</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>纬度：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit2_2">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>240</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>116.35</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit3_2">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>240</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>39.85</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit4_4">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>290</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label6_2">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>290</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>时</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn2">
    <property name="geometry">
     <rect>
      <x>662</x>
      <y>360</y>
      <width>281</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>计算矩形区域1内出租车数目</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit4_1">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>290</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QLabel" name="label7">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>290</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>日</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>450</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>分为r*r个格子,r=</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit5">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>450</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn3">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>450</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>确定</string>
    </property>
   </widget>
   <widget class="QLabel" name="label7_2">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>290</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>日</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit4_3">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>290</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn4">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>400</y>
      <width>291</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>计算往来矩形区域1与其他区域的车流量</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit2_3">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>510</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>116.45</string>
    </property>
   </widget>
   <widget class="QLabel" name="label4_3">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>550</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>右下角经度：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label4_4">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>510</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>左上角经度：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label5_3">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>550</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>纬度：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label3_2">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>480</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>矩形区域2：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label5_4">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>510</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>纬度：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit3_3">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>510</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>40</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit2_4">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>550</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>116.5</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit3_4">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>550</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>39.92</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn5">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>590</y>
      <width>301</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>计算往来矩形区域1,2的车流量</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>130</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>北京市中心经纬度大致范围：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>150</y>
      <width>291</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>116.2 -- 116.6, 39.8 -- 40.0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>330</y>
      <width>281</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>日范围：2 -- 8    时范围：0 -- 23</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>290</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>2008年2月</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn6">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>630</y>
      <width>291</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>查询往来矩形区域1,2的k条频繁路径</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit6">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>630</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>630</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>输入k:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn7">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>670</y>
      <width>301</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>查询往来区域1,2的通行时间最短路径</string>
    </property>
   </widget>
   <widget class="MyWidget2" name="widget2" native="true">
    <property name="geometry">
     <rect>
      <x>389</x>
      <y>619</y>
      <width>221</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
   <widget class="MyWidget4" name="widget4" native="true">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>580</y>
      <width>120</width>
      <height>80</height>
     </rect>
    </property>
   </widget>
   <widget class="MyWidget1" name="widget1" native="true">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>540</y>
      <width>120</width>
      <height>80</height>
     </rect>
    </property>
   </widget>
   <widget class="MyWidget5" name="widget5" native="true">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>660</y>
      <width>120</width>
      <height>80</height>
     </rect>
    </property>
   </widget>
   <widget class="MyWidget6" name="widget6" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>510</y>
      <width>120</width>
      <height>80</height>
     </rect>
    </property>
   </widget>
   <widget class="MyWidget7" name="widget7" native="true">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>690</y>
      <width>120</width>
      <height>80</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1121</width>
     <height>26</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QMenu" name="menu1">
    <property name="title">
     <string>功能选择</string>
    </property>
    <addaction name="action1"/>
    <addaction name="action2"/>
    <addaction name="action4"/>
    <addaction name="action5"/>
    <addaction name="action6"/>
    <addaction name="action7"/>
   </widget>
   <addaction name="menu1"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action1">
   <property name="text">
    <string>显示轨迹</string>
   </property>
  </action>
  <action name="action2">
   <property name="text">
    <string>区域范围查找</string>
   </property>
  </action>
  <action name="action3">
   <property name="text">
    <string>3</string>
   </property>
  </action>
  <action name="action4">
   <property name="text">
    <string>区域关联分析1</string>
   </property>
  </action>
  <action name="action5">
   <property name="text">
    <string>区域关联分析2</string>
   </property>
  </action>
  <action name="action6">
   <property name="text">
    <string>频繁路径分析</string>
   </property>
  </action>
  <action name="action7">
   <property name="text">
    <string>通信时间分析</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QAxWidget</class>
   <extends>QWidget</extends>
   <header>qaxwidget.h</header>
  </customwidget>
  <customwidget>
   <class>MyWidget2</class>
   <extends>QWidget</extends>
   <header>mywidget2.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MyWidget4</class>
   <extends>QWidget</extends>
   <header>mywidget4.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MyWidget1</class>
   <extends>QWidget</extends>
   <header>mywidget1.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MyWidget5</class>
   <extends>QWidget</extends>
   <header>mywidget5.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MyWidget6</class>
   <extends>QWidget</extends>
   <header>mywidget6.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MyWidget7</class>
   <extends>QWidget</extends>
   <header>mywidget7.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
